File: rb/utility/make_context.rb
# CheckEMailOrUsernameForADataBreach SDK utility: make_context
require_relative '../core/context'
module CheckEMailOrUsernameForADataBreachUtilities
  MakeContext = ->(ctxmap, basectx) {
    CheckEMailOrUsernameForADataBreachContext.new(ctxmap, basectx)
  }
end

File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/check-e-mail-or-username-for-a-data-breach-sdk/go/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}
